Modelica/Media/package.mo

```diff
@@ -2217,12 +2217,21 @@ package Examples
     Real m_flow_ext;
     Real der_p;
     Real der_T;
+  protected
+    parameter SI.AbsolutePressure p01 = 100000.0 "state.p at time 0";
+    parameter Real pRate1(unit = "Pa/s") = 0 "state.p rate of change";
+    parameter SI.Temperature T01 = 200 "state.T at time 0";
+    parameter SI.TemperatureSlope Trate1 = 1000 "state.T rate of change";
```

Comment:
Sholdn't it be `Tslope`1 instead of `Trate1`? (Just consider its unit.)

(and similar to `Trate2`.)

Reply:
In that case, I suppose also the descriptions should be updated, and I guess we should take a global look at things called _rate_ where the right convention would be to say _slope_.  Anyway, with this PR already being merged, I suggest opening a separate PR for getting the _slope_ vs _rate_ right (globally).

Reply:
Maybe @hubertus65 can decide.